.sync.yml

```diff
@@ -37,12 +37,15 @@ Gemfile:
       - gem: 'test-kitchen'
       - gem: 'kitchen-digitalocean'
       - gem: 'kitchen-docker'
-        git: 'https://github.com/mcgege/kitchen-docker'
-        branch: 'leap15'
```

Comment:
I think they still didn't release a new release/gem since my patch, so testing opensuse leap 15 will not work ... I'd like to implement integration tests for current os versions, then we'll need this. But maybe we can point directly to the git of kitchen-docker instead to my fork as this pr is already merged?